pybsn/switch.py

```diff
@@ -0,0 +1,86 @@
+class Switch(object): 
+    """Switch based API operations
+
+    """
+    def __init__(self, client):
+        self.client = client
+
+        # Default configurable values
+        self.name = None
+        self.dpid = None
+        self.fabric_role = None
+        self.leaf_group = None
+
+    @staticmethod
+    def get_switches(client, filter=None):
+
+        if filter:
```

Comment:
Hmm. The goal was for the user to be able to dynamically make use the Big DB's predicates. What if the user wants to filter switches by the fabric_role or leaf_group?